.github/pull_request_template.md

```diff
@@ -12,6 +13,14 @@
 <!--- This PR will be merged by any repository approver when it meets all the points in the checklist -->
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 
-- [ ] The commits and have meaningful messages; the author will squash them [after approval](https://github.com/opendatahub-io/opendatahub-community/blob/main/contributor-cheatsheet.md#:~:text=Usually%20this%20is%20done%20in%20last%20phase%20of%20a%20PR%20revision) or will ask to merge with squash.
+- [ ] The commits have meaningful messages; the author will squash them after approval or will ask to merge with squash.
```

Comment:
Since in upstream we have now the bot, I believe we could rephrase this as:

```suggestion
- [ ] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
```

Reply:
Yeah, that sounds about right.